backend/tests/integration-async/conftest.py

```diff
@@ -57,24 +58,12 @@ def clear_environment(app):
     from sqlalchemy_utils.functions import create_database, database_exists
 
     with app.app_context():
-        if not database_exists(str(db_.engine.url)):
-            create_database(str(db_.engine.url))
-        db_.create_all()
-        db_.session.rollback()
-        db_.session.remove()
-        all_tables = db_.metadata.tables
-        for table_name, table_object in all_tables.items():
-            db_.session.execute(f"ALTER TABLE {table_name} DISABLE TRIGGER ALL;")
-            db_.session.execute(table_object.delete())
-            db_.session.execute(f"ALTER TABLE {table_name} ENABLE TRIGGER ALL;")
-        db.session.commit()
-        _es = app.extensions["invenio-search"]
-        list(_es.delete(ignore=[404]))
-        list(_es.create(ignore=[400]))
+        clear_db(db_)
+        clear_es(es)
```

Comment:
I'd make them fixtures instead of passing objects around

Reply:
But then It would not work too well with async tests, as fixtures there are different :/
It's similar to invenio's `_es_delete/create_indexes` functions ;)